sys/include/walltime.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2025 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    sys_walltime    wall-clock time helper functions
```

Comment:
```suggestion
 * @defgroup    sys_walltime    Wall-Clock Time Helper Functions
```

This looks nicer in the documentation